sql-prepared-plan-cache.md

```diff
@@ -6,6 +6,13 @@ aliases: ['/tidb/dev/sql-prepare-plan-cache']
 
 # SQL Prepared Execution Plan Cache
 
+> **Warning:**
+>
+> If a cached `Update` or `Delete` statement encounters a DDL operation that modifies the relevant schema during execution, it may cause data inconsistency between tables and indexes, leading to incorrect results ([#51407](https://github.com/pingcap/tidb/issues/51407)). Please pay attention to the progress of the fix and upgrade to the latest LTS version to resolve this issue. Possible workarounds before upgrading:
+> 
+> - Temporarily [disable the execution plan cache for Prepare statements](/system-variables.md#tidb_enable_prepared_plan_cache-new-in-v610) before submitting DDL, and restore the setting of plan cache after DDL execution.
+> - Avoid executing DDL during business hours. Immediately after executing DDL, run [`admin check table`](/sql-statements/sql-statement-admin-check-table-index.md) to verify the consistency between tables and indexes. If errors are found, rebuild the relevant indexes.
```

Comment:
```suggestion
> If a cached `UPDATE` or `DELETE` statement encounters a DDL operation that modifies the relevant schema during execution, it might cause data inconsistency between tables and indexes, leading to incorrect results. For more information, see [Issue #51407](https://github.com/pingcap/tidb/issues/51407). Keep updated to the progress of the latest LTS version to resolve this issue. The following are possible workarounds before upgrading:
> 
> - Temporarily [disable the execution plan cache for Prepare statements](/system-variables.md#tidb_enable_prepared_plan_cache-new-in-v610) before submitting DDL, and restore the setting of plan cache after the DDL execution.
> - Avoid executing DDL during business hours. Immediately after executing DDL, run [`ADMIN CHECK TABLE`](/sql-statements/sql-statement-admin-check-table-index.md) to verify the consistency between tables and indexes. If errors are found, rebuild the relevant indexes.
```